config/config.go

```diff
@@ -375,6 +375,9 @@ type ScrapeConfig struct {
 	Scheme string `yaml:"scheme,omitempty"`
 	// More than this many samples post metric-relabelling will cause the scrape to fail.
 	SampleLimit uint `yaml:"sample_limit,omitempty"`
+	// More than this many targets post config-relabelling will cause the scrape
+	// group to be dropped.
```

Comment:
I don't believe that either "scrape group" or "config relabelling" are terms we use.

Reply:
"relabeling phase" seems to be it.